recipes/hwm/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "hwm" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/hwm-{{ version }}.tar.gz
+  sha256: 5f09159edf703390f4d3110d4737b2f2c47754a8bf11c2b0993e67270773f8c8
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+
+requirements:
+  host:
+    - python >=3.9
+    - pip
+    - setuptools
+  run:
+    - python >=3.9
+    - numpy <2
+    - scipy >=1.7.0
+    - scikit-learn >=1.2.0
+    - pandas >=1.3.0
+    - joblib >=1.2.0
+    - matplotlib-base >=3.5.2  
+    
+test:
+  imports:
+    - hwm
+    - hwm.api
+    - hwm.compat
+    - hwm.datasets
+    - hwm.estimators
+    - hwm.utils
+  commands:
+    - python -c "import hwm; print(hwm.__version__)"
+  requires:
+    - python >=3.9
```

Comment:
Please follow the convention detailed in:
https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python

for setting `python_min` and how they are applied in each of the requirements sections.

Reply:
I fix it :
```yaml
{% set name = "hwm" %}
{% set version = "1.0.1" %}

package:
  name: {{ name|lower }}
  version: {{ version }}

source:
  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/hwm-{{ version }}.tar.gz
  sha256: 5f09159edf703390f4d3110d4737b2f2c47754a8bf11c2b0993e67270773f8c8

build:
  noarch: python
  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
  number: 0

requirements:
  host:
    - python >=3.9
    - pip
  run:
    - python >=3.9
    - numpy <2
    - scipy >=1.7.0
    - scikit-learn >=1.2.0
    - pandas >=1.3.0

test:
  imports:
    - hwm
  commands:
    - pip check
  requires:
    - pip

about:
  home: https://github.com/earthai-tech/hwm
  summary: Adaptive Hammerstein-Wiener Modeling Toolkit
  license: BSD-3-Clause
  license_file: LICENSE

extra:
  recipe-maintainers:
    - earthai-tech
    
```